### PR TITLE
Feature/analytic improvements

### PR DESCRIPTION
**Pull request by KirillMakarov**
**YouTrack task**: [#APPS-1638](https://vyahhi.myjetbrains.com/youtrack/issue/APPS-1638)

**Description List**:
* remove a lot of false positive errors, when internet is not enabled. (It is not error, it is expected state)
* rename some rarely used events to more meaningful name
* fix extra memory consumption in `SearchQueriesAdapter`

### CODE REVIEW

File: app/src/main/java/org/stepic/droid/base/CoursesDatabaseFragmentBase.java [hunk 1]
```diff
@@ -15,7 +15,6 @@
 import org.stepic.droid.analytic.Analytic;
 import org.stepic.droid.core.dropping.contract.DroppingListener;
 import org.stepic.droid.core.presenters.PersistentCourseListPresenter;
-import org.stepic.droid.model.Course;
```
Comment (eadm) [inline app/src/main/java/org/stepic/droid/base/CoursesDatabaseFragmentBase.java]: Why was this import removed? `org.stepic.droid.model.Course course` constructions look strange.

File: app/src/main/java/org/stepic/droid/ui/fragments/CourseDetailFragment.java [hunk 2]
```diff
@@ -215,22 +207,10 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         //VIEW:
         coursePropertyList = new ArrayList<>();
         joinCourseSpinner = new LoadingProgressDialog(getActivity());
-        footer = ((LayoutInflater) getActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE)).inflate(R.layout.fragment_course_detailed_footer, coursePropertyListView, false);
-        coursePropertyListView.addFooterView(footer);
-        instructorsCarousel = ButterKnife.findById(footer, R.id.instructors_carousel);
-
-        header = ((LayoutInflater) getActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE)).inflate(R.layout.fragment_course_detailed_header, coursePropertyListView, false);
-        coursePropertyListView.addHeaderView(header);
+        LayoutInflater layoutInflater = LayoutInflater.from(getContext());
```
Comment (eadm) [inline app/src/main/java/org/stepic/droid/ui/fragments/CourseDetailFragment.java]: Why not just [`getLayoutInflater()`](https://developer.android.com/reference/android/support/v4/app/Fragment.html#getLayoutInflater())?

Comment (KirillMakarov) [reply app/src/main/java/org/stepic/droid/ui/fragments/CourseDetailFragment.java]: right
